response_operations_ui/templates/admin/user-create.html

```diff
@@ -0,0 +1,311 @@
+                            "classes": "ons-u-mb-l",
+                            "title":  errorTitle
+                        })
+                %}
+                    {% for message in messages %}
+                        <p id="flashed-message-{{ loop.index }}">{{ message }}</p>
+                    {% endfor %}
+                {% endcall %}
+            {% endif %}
+        {% endwith %}
+          {% if form.errors %}
+            {% set errorData = [] %}
+                {% set error_count = form.errors | length %}
+                {% set plural_suffix = 's' if error_count > 1 else '' %}
+                {% set header_text = 'This page has ' ~ error_count ~ ' error' ~ plural_suffix %}
+            {% call
+                onsPanel({
+                    "title": header_text,
+                    "type": "error"
+                })
+            %}
+                <p>These <strong>must be corrected</strong> to continue.</p>
+                {% for error in form.first_name.errors %}
+                    {% do errorData.append(
+                        {
+                            "text": error,
+                            "index": true,
+                            "url": "#first_name",
+                            "classes": "ons-js-inpagelink"
+                        }
+                    ) %}
+                {% endfor %}
+                {% for error in form.last_name.errors %}
+                    {% do errorData.append(
+                        {
+                            "text": error,
+                            "index": true,
+                            "url": "#last_name",
+                            "classes": "ons-js-inpagelink"
+                        }
+                    ) %}
+                {% endfor %}
+                {% for error in form.email.errors %}
+                    {% do errorData.append(
+                        {
+                            "text": error,
+                            "index": true,
+                            "url": "#email",
+                            "classes": "ons-js-inpagelink"
+                        }
+                    ) %}
+                {% endfor %}
+                {{
+                    onsList({
+                        "classes": "list--bare",
+                        "itemsList": errorData
+                    })
+                }}
+            {% endcall %}
+        {% endif %}
+            <form method="post" action="" class="form" role="form">
+                <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+                <h1 class="ons-u-mt-l">Create your account</h1>
+
+                    {{
+                        onsInput({
+                            "id": "first_name",
+                            "name": "first_name",
+                            "type": "text",
+                            "label": {
+                                "description": "Enter the new users first name",
```

Comment:
Not convinced you need these descriptions

Reply:
It was part of the design 🤷‍♂️

Reply:
No worries, just seemed a bit overkill